src/maxdiffusion/configs/base_flux_dev.yml

```diff
@@ -189,17 +189,17 @@ checkpoint_every: -1
 enable_single_replica_ckpt_restoring: False
 
 # Training loop
-learning_rate: 4.e-7
+learning_rate: 1.e-5
```

Comment:
I think these can be left unchanged in this PR and can be modified in the training PR.